website/content/en/docs/concepts/disruption.md

```diff
@@ -84,6 +84,10 @@ spec:
 ```
 {{% /alert %}}
 
+{{% alert title="Warning" color="warning" %}}
```

Comment:
Can we make sure to add this change to the other versions of the docs? Specifically, `preview`, since this is the version of the docs that we take forward from version to version